roles/docker/tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
 # This role contains tasks for configuring and starting docker service
 
+# XXX: figure a way to download a specific docker version
```

Comment:
thanks for the pointer @erikh let me look at the changes required to use a docker release.

Since we already modify the systemd units for docker it might not be that complex to download the release and set it up ourselves.